docs/CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   FlawCVSS, FlawReference, Snippet and Tracker. (OSIDB-3466)
 - Moved metadata creation during tests to root level conftest and
   automatically set envs during VCR recording (OSIDB-3492)
+- Implement complex logic in workflow state requirements and allow moving a flaw to
+  state done if it has no trackers but impact is moderate or low (OSIDB-3524)
```

Comment:
I have just a very small nitpick, I would probably split this up and move implementation of the complex condition logic into `Added` section and in `Changed` section I would leave the workflow `DONE` state conditions change. But might be just my personal preference, leaving it up to you.